Added discontinued note

Dan0xE opened the pull request.

scoopscode commented on aircraft.html:
```diff
@@ -0,0 +1,45 @@
+			
+    </div>
+    
+    <h1 class="title">AIRCRAFT UPDATES</h1>
```
Change "AIRCRAFT UPDATES" to "UPDATES" for all progress from our projects.

scoopscode commented on aircraft.html:
```diff
@@ -0,0 +1,45 @@
+    </div>
+    
+    
+    <footer class="copy"> ©️ Pinpoint Simulations 2021</footer>
```
Remove copyright (unless we are copyrighted in which I didnt know).

Spherrrical replied:
We are allowed copyright under the GPLv3 License. You can check description for their copyright notice.

scoopscode commented on index.html:
```diff
@@ -17,6 +17,7 @@
       <a class="activeClass"  href="index.html">HOME</a>
       <a class="navbarlink" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Same from above, change to "UPDATES"

scoopscode commented on privacy.html:
```diff
@@ -18,6 +18,7 @@
       <a class="navbarlink" href="index.html">HOME</a>
       <a class="navbarlink" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Same as above, change to "UPDATES"

scoopscode commented on pull_requests.html:
```diff
@@ -17,6 +17,7 @@
       <a class="navbarlink" href="index.html">HOME</a>
       <a class="activeClass" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Same as above, change to "UPDATES"

scoopscode commented on tos.html:
```diff
@@ -16,6 +16,7 @@
       <a class="navbarlink" href="index.html">HOME</a>
       <a class="navbarlink" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Same as above, change to "UPDATES"

Spherrrical commented:
Everything has been changed to current

Dan0xE committed:
Added discontinued note

Dan0xE committed:
Removed pages

Dan0xE committed:
Unused pages / CSS removed

Dan0xE committed:
Removed Element

Dan0xE committed:
Cleaned a bit up

- Removed link to home in Navbar
- Removed TOS since its not needed
- Removed reddit since it links to and individual instead of the org
- Cleaned up css/img
- Cleaned up code and refactoring
